apps/chart-docsite/.eslintrc.json

```diff
@@ -4,7 +4,7 @@
   "rules": {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33527&commitId=05eae6ad8fa0a83258a3a24ce097f08b80c89db3&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Right center.chromium.png|2396| Changed|
|Callout.Rendering callout attached to a rectangle.chromium.png|1493| Changed|
|Callout.Gap space 25.chromium.png|2309| Changed|

</details>


  <details><summary><b>Panel</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Panel.SearchBox and Right Panel.chromium.png|1611865| Changed|
|Panel.Search Box And Right Panel RTL.chromium.png|1611865| Changed|

</details>


  <details><summary><b>SearchBox</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox.Root.chromium.png|1612584| Changed|
|SearchBox.Show Icon RTL.chromium.png|1612542| Changed|
|SearchBox.Root RTL.chromium.png|1612472| Changed|
|SearchBox.ShowIcon.chromium.png|1612542| Changed|
|SearchBox.Full RTL.chromium.png|1612583| Changed|
|SearchBox.Full.chromium.png|1612583| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic.chromium.png|376| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>